docs/usage.md

```diff
@@ -30,74 +36,92 @@ For more specific integration options, please check out the [integration documen
 
 ## Understanding the output format
 
-TODO
+The output format is designed for you to use the generated models in further contexts. It might be part of a larger code generation such as AsyncAPI templates. This means that you can glue multiple models together into one large file, or split it out as you see fit.
 
 ## Generate models from AsyncAPI documents
 
-When providing an AsyncAPI document, the library iterates the entire document and generate models for all defined messages. If any other kind of iteration is wanted, feel free to create a [feature request](https://github.com/asyncapi/modelina/issues/new?assignees=&labels=enhancement&template=enhancement.md).
+When providing an AsyncAPI document, the library iterates the entire document and generate models for all defined message payloads. The message payloads are interpreted based on the schema format. 
```

Comment:
We should clarify it that custom formats are converted to the JSON Schema, and then that JSON Schema follows the JSON Schema input processing.

Reply:
Technically, we should not convert the payload to JSON Schema, but instead determine based on the schemaFormat which other input processor we should use to process the message payload 🤔 I think I left it out on purpose for this change 🧐

Reply:
So then you will have to write your own logic to handle custom schemas and modelina will use parser-js only to check spec if is valid 🤔

Reply:
Basically yes. 

We still yet have to see how we handle non-json payloads, cause it might happen we have to do some kind of handling there. But otherwise yes not much the AsyncAPI input processor needs to do 😄